src/main/java/ladder/model/Ladder.java

```diff
@@ -0,0 +1,72 @@
+package ladder.model;
+
+import java.util.stream.IntStream;
+import ladder.dto.LineDto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Ladder {
+    private static final Random random = new Random();
```

Comment:
프로젝트 전반에 걸쳐 상수 네이밍 컨벤션을 전체 대문자로 해보는건 어떨까요?

Reply:
오... 한번도 상수화된 인스턴스를 상수라고 생각해 본 적이 없었는데 생각해보니 상수가 아닐 이유가 없네요.

상수 변수에 대해서는 전체 대문자 네이밍 컨벤션을 지키는 편인데, 이와 같은 경우는 사람마다 의견이 갈리는 것 같아요.

https://stackoverflow.com/questions/7259687/java-naming-convention-for-static-final-variables/7259738

로거를 `LOGGER`로 쓸지 `logger`로 쓸지 결정하는 것이 비슷한 상황일 것 같은데, 저는 소문자가 편한 것 같아요!

피케이는 `static final` 로 선언된 인스턴스 변수들도 모두 uppercase로 작성하시나요?

Reply:
크루들과 이야기해보다가 이런 글을 발견했어요!
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

```java

// Constants
static final int NUMBER = 5;
static final ImmutableList<String> NAMES = ImmutableList.of("Ed", "Ann");
static final Map<String, Integer> AGES = ImmutableMap.of("Ed", 35, "Ann", 32);
static final Joiner COMMA_JOINER = Joiner.on(','); // because Joiner is immutable
static final SomeMutableType[] EMPTY_ARRAY = {};

// Not constants
static String nonFinal = "non-final";
final String nonStatic = "non-static";
static final Set<String> mutableCollection = new HashSet<String>();
static final ImmutableSet<SomeMutableType> mutableElements = ImmutableSet.of(mutable);
static final ImmutableMap<String, SomeMutableType> mutableValues =
    ImmutableMap.of("Ed", mutableInstance, "Ann", mutableInstance2);
static final Logger logger = Logger.getLogger(MyClass.getName());
static final String[] nonEmptyArray = {"these", "can", "change"};
```